docs/reference/proto-design-guidelines.md

```diff
@@ -144,13 +144,13 @@ Here is a simple example of a good PROTO declaration (the implementation is not
 # license url: http://www.apache.org/licenses/LICENSE-2.0
 # A color pencil
 
-ColorPencil {
+ColorPencil [
   SFFloat    translation          0 0 0
```

Comment:
To be valid the we should also add the `field` keyword in each line.
(This field type was also wrong)
```suggestion
  field SFVec3f    translation          0 0 0
```